docs/quickstart/java_api_quickstart.md

```diff
@@ -46,39 +46,79 @@ Use one of the following commands to __create a project__:
     {% highlight bash %}
     $ mvn archetype:generate                               \
       -DarchetypeGroupId=org.apache.flink              \
-      -DarchetypeArtifactId=flink-quickstart-java      \
+      -DarchetypeArtifactId=flink-quickstart-java      \{% unless site.is_stable %}
+      -DarchetypeCatalog=https://repository.apache.org/content/repositories/snapshots/ \{% endunless %}
       -DarchetypeVersion={{site.version}}
     {% endhighlight %}
         This allows you to <strong>name your newly created project</strong>. It will interactively ask you for the groupId, artifactId, and package name.
     </div>
     <div class="tab-pane" id="quickstart-script">
     {% highlight bash %}
+{% if site.is_stable %}
     $ curl https://flink.apache.org/q/quickstart.sh | bash
+{% else %}
+    $ curl https://flink.apache.org/q/quickstart-SNAPSHOT.sh | bash
+{% endif %}
     {% endhighlight %}
     </div>
 </div>
 
 ## Inspect Project
 
-There will be a new directory in your working directory. If you've used the _curl_ approach, the directory is called `quickstart`. Otherwise, it has the name of your artifactId.
+There will be a new directory in your working directory. If you've used
+the _curl_ approach, the directory is called `quickstart`. Otherwise,
```

Comment:
yes, that's what it does (internally) but wouldn't you say, a user not knowing these sees the two ways of creating the repo as two "approaches" and would be rather confused calling it a "shortcut"? - it could be made explicit but then we could argue why the script is needed in the first place... (the user will copy the command anyway)
however I did not change that and did not intend to